app/src/main/java/de/tum/in/tumcampusapp/fragments/PdfViewFragment.java

```diff
@@ -37,7 +40,32 @@ public void onResume() {
         pdfView.fromFile(pdf).load();
     }
 
-    public void setPdf(File pdf){
-        this.pdf = pdf;
+    public boolean setPdf(File pdf){
+        if (isFilePdf(pdf)){
+            this.pdf = pdf;
+            return true;
+        }
+        return false;
+    }
+
+    private boolean isFilePdf(File file){
```

Comment:
@t2m4k1 Solid research 👍 

But please document that in the function javadoc, so that knowledge doesn't get lost.

Also: please use the `Utils.log()` methods instead of `e.printStackTrace()` or debugging becomes hell